Source/Particles/Collision/BinaryCollision/VirtualPhotonCreation.H

```diff
@@ -0,0 +1,244 @@
+/* Copyright 2021 Neil Zaim
```

Comment:
I have expressed my preference for removing the list of authors at the top of the files a few times in the past years, lastly in https://github.com/BLAST-WarpX/warpx/pull/6102#discussion_r2294962438 (in case you have an opinion and want to chime in), but since the majority of developers prefer to keep it or don't have a preference, I suggest to fix it in this case:
```suggestion
/* Copyright 2025 Arianna Formenti
```